semgrep/semgrep/error.py

```diff
@@ -162,13 +162,18 @@ def _stack_trace(self) -> str:
         """
         if self.error_type == "Fatal error":
             error_trace = self.details or "<no stack trace returned>"
-            return f"\n-----[ BEGIN error trace ]-----\n{error_trace}\n-----[ END error trace ]-----\n"
+            return f"\nSemgrep failed to run this rule.\nWe don't know yet why this happened, but happy to help. Please file a GitHub issue or report it on Slack.\n\n====[ BEGIN error trace ]====\n{error_trace}=====[ END error trace ]=====\n"
```

Comment:
Can we make this more actionable by providing a link? Modify/re-use PLEASE_FILE_ISSUE_TEXT from semgrep/semgrep/constants.py?

Reply:
Definitely, I didn't know about